### ISSUE TITLE
MP3 Playback not working

### ISSUE THREAD

**Issue by frakman1** [coderaiser/cloudcmd#322]
* **Version** (`cloudcmd -v`): v15.5.1
* **Node Version** `node -v`: v15.7.0
* **OS** (`uname -a` on Linux): Linux 70504fae93ac 4.19.107-Unraid #1 SMP Thu Mar 5 13:55:57 PST 2020 x86_64 GNU/Linux
* **Browser name/version**: Chome Version 88.0.4324.96 (Official Build) (x86_64)
* **Used Command Line Parameters**: 
```bash
docker run -d --name='CloudCommander' --net='bridge' -e TZ="America/New_York" -e HOST_OS="Unraid" -e 'HOME'='/root' -p '8765:8000/tcp' -v '/<my-path>/':'/UNRAID':'rw' -v '/mnt/user/appdata/cloudcmd':'/root':'rw' -u 99:100 'coderaiser/cloudcmd'
```

When I select an mp3 file and click on "View" it shows some scrolling animation next to the file name but doesn't actually play anything.

I went back to an earlier release `coderaiser/cloudcmd:14.9.3-x64`, it worked normally again with a popup and play button.
Looks like a regression issue.

**Comment by coderaiser** [comment]
Thanks a lot :), fixed with 3cd1d67, landed in [v15.5.2](https://github.com/coderaiser/cloudcmd/releases/tag/v15.5.2) 🎉 .
Is it works for you?

**Comment by frakman1** [comment]
Yes, that works. Thank you for the quick resolution.
